tools/deb/BUILD

```diff
@@ -0,0 +1,51 @@
+load("@bazel_tools//tools/build_defs/pkg:pkg.bzl", "pkg_tar", "pkg_deb")
+
+pkg_tar(
+    name = "envoy-bin",
+    srcs = [
+        "istio-iptables.sh",
+        "istio-start.sh",
+        "//src/envoy/mixer:envoy",
+    ],
+    mode = "0755",
+    package_dir = "/usr/local/bin",
```

Comment:
we shouldn't have hardcoded paths (expect /usr/local/bin/...) anywhere - do we ? but agreed we can start somewhere and TODO the rest (same question as below for /etc/istio vs /var/lib/...)